src/Apps/Order/Routes/__tests__/Shipping.jest.tsx

```diff
@@ -1089,6 +1090,16 @@ describe("Shipping", () => {
           },
         })
       })
+      it.only("set shippinglike above with trcking", async () => {
+        renderWithRelay({
+          CommerceOrder: () => order,
+          Me: () => meWithAddresses,
+        })
+
+        await saveAndContinue()
+
+        expect(mockTrackEvent.mock.calls).toEqual({})
+      })
```

Comment:
forgot to remove this. I was discussing with @rquartararo how we might backfill tests for analytics that are missing as part of the shipping2 work - write a failing test, see what actually gets tracked, verify it makes sense, copy into old test, copy into new test.